pyproject.toml

```diff
@@ -10,7 +10,7 @@ classifiers = [
     "License :: OSI Approved :: MIT License",
     "Operating System :: OS Independent",
 ]
-dependencies = ["click>=8.1.7", "pygithub>=2.1.1", "click-spinner>=0.1.10", "gitpython>=3.1", "semver>=3.0", "Jinja2>=3.1.3", "ruamel.yaml>=0.17.32", "boto3>=1.34.135"]
+dependencies = ["click>=8.1.7", "pygithub>=2.1.1", "click-spinner>=0.1.10", "gitpython>=3.1", "semver>=3.0", "Jinja2>=3.1.3", "ruamel.yaml>=0.17.32", "boto3>=1.34.135", "pyyaml>=6.0.1"]
```

Comment:
`pyyaml` is unnecessary here, we already have `ruamel.yaml`. `pyyaml` does not support YAML 1.2 (released in 2009) that fixes the vast majority of the complaints people had about the edge cases in the YAML spec ("NO" -> "Norway" is a classic example).

Please update the magicdust code you've brought over to use `ruamel.yaml`.